h/streamer/app.py

```diff
@@ -46,6 +46,7 @@ def create_app(_global_config, **settings):
         {
             "h_pyramid_sentry.filters": SENTRY_ERROR_FILTERS,
             "h_pyramid_sentry.celery_support": True,
+            "h_pyramid_sentry.sqlalchemy_support": True,
```

Comment:
Also enabling Sentry's SQLAlchemy support. This is for Sentry crash reporting, not logs. The main h app has it so I don't see any reason why the WebSocket app shouldn't.